cli/cmd/plugin/ui/web/tanzu-ui/src/components/VSphereAuthForm.tsx

```diff
@@ -24,34 +26,52 @@ function VSphereAuthForm () {
             payload: data
         });
     };
+
+    const { currentStep, setTabStatus, tabStatus } = props;
+    useTabStatus(errors, { currentStep, setTabStatus, tabStatus });
+
+    const blurHandler = () => {
```

Comment:
question, should blurHandler maybe go in the form.service?

Reply:
moved